src/entity/src/user.js

```diff
@@ -785,18 +807,40 @@ export class User {
    * @return {Promise<Object>} The response.
    */
   async forgotUsername(options = {}) {
```

Comment:
Yea I like that idea. Would you like me to add deprecation documentation for this release?